projects/packages/videopress/src/class-data.php

```diff
@@ -31,9 +31,11 @@ public static function get_blog_id() {
 	 */
 	public static function get_videopress_videos_private_for_site() {
 		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-			return boolval( get_blog_option( get_current_blog_id(), 'videopress_private_enabled_for_site', false ) );
+			$blog_is_private = ( intval( get_blog_option( get_current_blog_id(), 'blog_public', '' ) ) === -1 );
+			return boolval( get_blog_option( get_current_blog_id(), 'videopress_private_enabled_for_site', $blog_is_private ) );
 		} else {
-			return boolval( get_option( 'videopress_private_enabled_for_site', false ) );
+			$blog_is_private = ( intval( get_option( 'blog_public', '' ) ) === -1 );
```

Comment:
I guess `blog_public` is not defined for Jetpack sites, right? so `$blog_is_private` will be False.

Reply:
We may want to priorize the site privacy when the site is private. Or, we should inform clearly to the users about this change in the module section.

<img width="574" alt="image" src="https://user-images.githubusercontent.com/77539/219364900-e461e00a-70f2-43fb-a806-1c26c5b62504.png">

Users that think their videos are private because the site is too, and realizes they don't because they changed the setting from the VideoPress module section could be a pain.